packages/reporter/cypress/support/component.ts

```diff
@@ -0,0 +1,8 @@
+import { mount } from 'cypress/react'
+import 'cypress-real-events/support'
+import { installCustomPercyCommand } from '@packages/ui-components/cypress/support/customPercyCommand'
+
+import '../../src/main.scss'
+
+Cypress.Commands.add('mount', mount)
```

Comment:
You mentioned in slack there was an issue with 
```
declare global {
  namespace Cypress {
    interface Chainable {
      mount: typeof mount
    }
  }
}
```

I added this and included `cypress/support/component.ts` in the tsconfig and everything compiled fine, plus it removed some of the ts warnings.

Reply:
The issue with that was because I wasn't using our webpack configs (and therefor everything was broken, that just being one example). I'll add the type addition back in.